app/models/movie.rb

```diff
@@ -1,3 +1,10 @@
 class Movie < ApplicationRecord
   belongs_to :studio
+  has_many :actor_movies
+  has_many :actors, through: :actor_movies
 end
+
+def average_age
+  actors.average(:age)
```

Comment:
Nice! This is what I meant in the previous comment! This will grab actors for a given movie rather than all actors